ae_page_objects.gemspec

```diff
@@ -5,7 +5,7 @@ Gem::Specification.new do |s|
   s.name                      = "ae_page_objects"
   s.version                   = AePageObjects::VERSION
 
-  s.required_ruby_version     = '>= 1.9.3'
+  s.required_ruby_version     = '>= 2.2.5'
   s.authors                   = ["Donnie Tognazzini"]
```

Comment:
Update this line?

Reply:
Good idea, changed to "AppFolio Engineering"

Reply:
👍